service/controller/setting/types.go

```diff
@@ -53,3 +54,10 @@ type OIDC struct {
 	UsernameClaim string
 	GroupsClaim   string
 }
+
+type AzureNetwork struct {
+	CalicoSubnetMask int
+	MasterSubnetMask int
+	VPNSubnetMask    int
+	WorkerSubnetMask int
+}
```

Comment:
I don't think it's needed anymore.

Reply:
Why ? This is actually used by `network.Compute` to determine the size of subnets

Reply:
For the record we discussed those changes in a call. Changes incoming.

Reply:
done

dropped flags and using constants now